### PR TITLE
8844: Columns and Rows fields do not have limits

### PR DESCRIPTION
**Pull request by savvinsergey**

### CODE REVIEW

File: src/chicletSlicer.ts [hunk 1]
```diff
@@ -618,10 +618,18 @@ module powerbi.extensibility.visual {
                 ? 0
                 : data.slicerSettings.general.columns;
 
+            data.slicerSettings.general.columns = data.slicerSettings.general.columns > 1000
+                ? 1000
```
Comment (ignatvilesov) [inline src/chicletSlicer.ts]: Please move ```1000``` to a static variable. We shouldn't use magic numbers.

File: src/chicletSlicer.ts [hunk 2]
```diff
@@ -618,10 +618,18 @@ module powerbi.extensibility.visual {
...
+                : data.slicerSettings.general.columns;
+
             data.slicerSettings.general.rows = data.slicerSettings.general.rows < 0
                 ? 0
                 : data.slicerSettings.general.rows;
 
+            data.slicerSettings.general.rows = data.slicerSettings.general.rows > 1000
+                ? 1000
```
Comment (ignatvilesov) [inline src/chicletSlicer.ts]: Please move ```1000``` to a static variable. We shouldn't use magic numbers.

Comment (savvinsergey) [inline src/chicletSlicer.ts]: Done

File: src/chicletSlicer.ts (hunk 1 above)
Comment (savvinsergey) [inline src/chicletSlicer.ts]: Done

**Comment by savvinsergey** [comment]
@AviSander I guess we can't do it via capabilities.json

**Pull request merged**